applications/API/schema_extenstions.py

```
@@ -0,0 +1,33 @@
+from rest_framework import serializers
+from drf_spectacular.extensions import OpenApiViewExtension
+from drf_spectacular.utils import extend_schema, inline_serializer, OpenApiResponse
+
+class DjoserUserDeleteExtension(OpenApiViewExtension):
+    # Target the Djoser view (import path can vary by version)
+    # target_class = "djoser.views.UserViewSet"SA
+
+    def view_replacement(self):
+        from djoser.views import UserViewSet
+
+        class PatchedUserViewset(UserViewSet):
```

Pylint (3.3.7)

[warning] Unused variable 'PatchedUserViewset'

(W0612)